qa/integration-tests/src/test/java/io/camunda/zeebe/it/config/IdleStrategyConfigIT.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.it.config;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.zeebe.qa.util.cluster.TestStandaloneGateway;
+import io.camunda.zeebe.shared.IdleStrategyConfig.IdleStrategySupplier;
+import java.time.Duration;
+import org.junit.jupiter.api.Test;
+
+final class IdleStrategyConfigIT {
+  @Test
+  void shouldConfigureIdleStrategy() {
+    // given
+    try (final var gateway = new TestStandaloneGateway()) {
+      gateway
+          .withProperty("zeebe.actor.idle.maxSpins", 50L)
+          .withProperty("zeebe.actor.idle.maxYields", 62L)
+          .withProperty("zeebe.actor.idle.minParkPeriod", Duration.ofNanos(100))
+          .withProperty("zeebe.actor.idle.maxParkPeriod", Duration.ofNanos(500));
+
+      // when
+      gateway.start();
+
+      // then
+      final var idleStrategy = gateway.bean(IdleStrategySupplier.class).get();
```

Comment:
💭 Do we have the supplier to get it for tests?

Reply:
What do you mean? This is getting the supplier in a test.